### PR TITLE
Chromium safe-search

### PR DESCRIPTION
**Pull request by skarbat**
* Disabling google.com when setting parental control to "high"
 * Until we have a higher version of Chromium that supports this sestting in policy files

### CODE REVIEW

File: kano_settings/system/advanced.py [hunk 1]
```diff
@@ -368,5 +368,14 @@ def set_parental_level(level_setting):
         set_ultimate_parental(False)
 
     blacklist, whitelist = read_listed_sites()
+
+    # SafeSearch: disabling google.com, until we have higher Chromium version
+    # that implements policy files.
+    if 'chromium' in enabled:
+        if blacklist:
```
Comment (alex5imon) [inline kano_settings/system/advanced.py]: I don't think we should do this... 

There's a function to write to the blacklist:
https://github.com/KanoComputing/kano-settings/blob/safe-search/kano_settings/system/advanced.py#L339

also blacklist should exist, if it doesn't then we have a problem we shouldn't just create the list with only 'google.com'

Also we need add more:
www.google.com
google.com
www.google.uk
google.uk
www.google.es
google.es
www.google.de
google.de

and so on...

Comment (alex5imon) [inline kano_settings/system/advanced.py]: check this line for the 'www' thing: https://github.com/KanoComputing/kano-settings/blob/master/kano_settings/system/advanced.py#L142-L149

**Comment by skarbat** [comment]
Changed the way we disable google searches. I tested it and seems to work pretty well. @alex5imon

**Comment by alex5imon** [comment]
@tombettany could you review this please ??

**Comment by alex5imon** [comment]
@skarbat when testing this we will need to make sure the feedback widget still works (sending the feedback to the google doc)

**Comment by alex5imon** [comment]
s***t I didn't want to close this sorry

File: kano_settings/system/advanced.py [hunk 2]
```diff
@@ -15,6 +15,8 @@
 import signal
 import urllib2
 from bs4 import BeautifulSoup
+import pycountry
+import gzip
```
Comment (tombettany) [inline kano_settings/system/advanced.py]: Is the `gzip` module installed by default?

**Comment by tombettany** [comment]
Nice solution @skarbat - especially the transferring to using the `gzip` module. Apart from my above comments, I'm happy with this.

**Comment by skarbat** [comment]
That's correct @tombettany , installed on our image for Python 2.6, 2.7 and 3.2.

**Comment by tombettany** [comment]
Then @skarbat, please merge :+1:  :shipit:

**Pull request merged**